packages/g/grpc/xmake.lua

```diff
@@ -35,6 +35,7 @@ package("grpc")
 
     on_load(function (package)
         package:addenv("PATH", "bin")
+        package:add("deps", "abseil", {configs = {cxx_standard = "17"}})
```

Comment:
you can write it in the description scope using `add_deps("abseil", {configs = {cxx_standard = "17"}}})`